pkg/kncloudevents/http_client_new.go

```diff
@@ -29,29 +30,37 @@ import (
 )
 
 const (
-	defaultRetryWaitMin = 1 * time.Second
-	defaultRetryWaitMax = 30 * time.Second
+	defaultRetryWaitMin    = 1 * time.Second
+	defaultRetryWaitMax    = 30 * time.Second
+	defaultRecheckInterval = 5 * time.Minute
```

Comment:
```suggestion
	defaultCleanupInterval = 5 * time.Minute
```
I would include `cleanup` into the name, as this is what the routine does later. WDYT?